sdk/loadtesting/load-testing-rest/samples-dev/sample.ts

```diff
@@ -10,7 +10,7 @@
 
 import AzureLoadTesting, { isUnexpected, getLongRunningPoller } from "@azure-rest/load-testing";
 import { DefaultAzureCredential } from "@azure/identity";
-import { createReadStream } from "fs";
+import { createReadStream } from "node:fs";
 import { v4 as uuidv4 } from "uuid";
```

Comment:
nitpick: while you are at it, replace uuid with core-utils helper?

Reply:
Replaced all with Node core since they are Node samples such as `node:crypto` and `node:fs`